Content.Client/Animations/ReusableAnimations.cs

```diff
@@ -20,7 +21,8 @@ public static void AnimateEntityPickup(IEntity entity, EntityCoordinates initial
 
             if(!entity.TryGetComponent(out SpriteComponent sprite0))
             {
-                throw new Exception($"Entity doesn't have a {nameof(SpriteComponent)}!");
+                Logger.Warning($"Entity doesn't have a {0}!", nameof(SpriteComponent));
```

Comment:
sorry to bother you again mam but this should be an error imo, aswell as more descriptive of where it happens and whats specifically is failing. actually lemme just:
$"Entity couldn't be animated for pickup since it doesn't have a {0}!", nameof(SpriteComponent)